[PROOFREADING] Muun Wallet Indonesian Proofreading

MilankovicProtocol opened the pull request.

MilankovicProtocol committed:
Update id.md

Asi0Flammeus commented:
@MilankovicProtocol Thanks for your proofreading!

@MariJJhodl Can you confirm that everything went well and it is ready to be merge

MariJJhodl commented:
@MilankovicProtocol you added some bolds here and there, I saw. I will leave them, since they may enhance clarity, but please respect the initial format for the next proofreadings. Unless you send me some feedback on this kind of things, then we might change the format for all languages, if we think it is necessary.

Ready to be merged

MilankovicProtocol committed:
Update id.md

Hello, I apologize; there was a mistake on my end. The proofreading from lines 168-270 has not been included in this pull request before. I have fixed that, and now the proofreading is completed.

MilankovicProtocol commented:
Hello @MariJJhodl , thank you for the feedback

According to the Bahasa Indonesia written language standards, foreign terms should be italicized, which is why I italicized the English and other foreign words in this proofreading.
